sources/osg/Matrix.js

```diff
@@ -611,6 +649,22 @@ var Matrix = {
         };
     } )(),
 
+    preMultRotate: function ( matrix, quat ) {
+        if ( Quat.zeroRotation( quat ) )
+            return;
+        var r = Matrix.create();
```

Comment:
nop it does not seem fixed see https://github.com/cedricpinson/osgjs/blob/jtorresfabra/AutoTransform/sources/osg/Matrix.js#L640 to create the temp qtemp